ops/charm.py

```diff
@@ -297,6 +297,20 @@ def restore(self, snapshot):
             self.unit = None
 
 
+class RelationCreatedEvent(RelationEvent):
+    """Represents the `relation-created` hook from Juju.
+
+    It's triggered by `CharmBase.on[RELATION].relation_created`.
+
+    This is triggered whenever relations are created in the Juju model before relation-joined
+    events. For units that are coming up, relation-created events fire after the install event
+    but before any events, including leader-elected. This allows a leader unit to work with
+    peer application relation data early in its life cycle.
+    """
+
+    pass
```

Comment:
All of these "pass" should go away when we have docstrings.

Reply:
Agreed, thanks for noticing.

Reply:
https://github.com/canonical/operator/pull/218/commits/6e5916633b97995ca88a080597dff7c6e9ceff4c